docker/zarr-to-loom/optimus_zarr_to_loom.py

```diff
@@ -37,13 +35,16 @@ def main():
     # Open the ZARR
     store = zarr.DirectoryStore(input_zarr_path)
     root = zarr.open(store)
-
+    
+    #Get expression data type: exonic or whole_transcript
+    expression_data_type = root.attrs['expression_data_type']
+    
     # Get the expression matrix
     # expression matrix in numpy ndarray format (dense)
     # NOTE: If memory is limiting this could be done by chunk
-    nrows = root[f"/{sample_id}.zarr/expression"].shape[0]
```

Comment:
Although good simplifications these changes may be assuming different input and or otherwise be breaking the pipeline. Please investigate why the tests failed.

Reply:
The current error message on the failed Optimus run is:
 `Failed to process 'call ZarrUtils.OptimusZarrToLoom' (reason 1 of 1): The call supplied a value 'sample_id' that doesn't exist in the task (or sub-workflow))`